test/rules/base.jl

```diff
@@ -65,36 +82,46 @@ end
         # TODO: atan2 sincos
     end
     @testset "Misc. Tests" begin
-        @testset "*(x, y)" begin
-            x, y = rand(3, 2), rand(2, 5)
+        @testset "*(x, y)" for T in (Float32, Float64, Complex{Float64})
```

Comment:
```suggestion
        @testset "*(x::$T, y::$T)" for T in (Float32, Float64, Complex{Float64})
```
otherwise the name of the testset doesn't actually tell you what it's testing.

Reply:
Thanks for the suggestion! Will change that.